1-org/envs/shared/variables.tf

```diff
@@ -97,6 +97,12 @@ variable "log_export_storage_retention_policy" {
   default = null
 }
 
+variable "vpc" {
+  description = "The type of VPC to attach the project to. Possible options are base or restricted."
+  type        = string
+  default     = "null"
```

Comment:
I'm not sure why you are using `default = "null"`. If you want the default value would be `null`, it shouldn't use quotation marks. If you want the value to be empty, you should use `default = ""`.